### PR TITLE
added router component

### PR DESCRIPTION
**Pull request by nikitha2000**
added router component

### CODE REVIEW

File: src/component/Favourites/Favourites.jsx [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
+    const storedFavourites =
+      JSON.parse(localStorage.getItem("favourites")) || [];
+    setFavouritePhotos(storedFavourites);
+  }, []);
+
+  const handleToggleFavourite = (id) => {
+    setFavouritePhotos((prevPhotos) => toggleFavourite(prevPhotos, id));
+  };
+
+  if (!favouritePhotos || favouritePhotos.length === 0) {
```
Comment (vonnue-nidheesh) [inline src/component/Favourites/Favourites.jsx]: `if (!favouritePhotos?.length) {`

File: src/component/Gallery/GalleryLayout.jsx [hunk 2]
```diff
@@ -36,38 +36,25 @@ const Photos = () => {
...
 
   const handleNavItemSelect = (item) => {
     setSelectedNavItem(item);
   };
 
-  const sortedFavouritesPhotos = photos.filter((photo) => photo.favourites);
-
   return (
     <>
       <Navbar onNavItemSelect={handleNavItemSelect} />
```
Comment (vonnue-nidheesh) [inline src/component/Gallery/GalleryLayout.jsx:48]: Should we need nav item here?

File: src/component/Gallery/PhotoItem.jsx [hunk 3]
```diff
@@ -13,6 +13,15 @@ const PhotoItem = ({ photo, onToggleFavourite }) => {
     setIsHovered(false);
   };
 
+  const handleClick = () => {
+    console.log("Photo ID:", photo.id);
```
Comment (vonnue-nidheesh) [inline src/component/Gallery/PhotoItem.jsx]: Remove console.

File: src/component/Gallery/toggleFavourite.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const toggleFavourite = (photos, id) => {
```
Comment (vonnue-nidheesh) [inline src/component/Gallery/toggleFavourite.jsx]: in this file, there is only a function right. So can you change this file name as `toggleFavourite.js`

File: src/component/Gallery/GalleryLayout.jsx (hunk 2 above)
Comment (nikitha2000) [reply src/component/Gallery/GalleryLayout.jsx:48]: no i deleted it later

**Review by vonnue-nidheesh — APPROVED** [review]

**Pull request merged**